serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

```diff
@@ -23,6 +23,11 @@ A window appears containing your current operating system as selected and a link
 --
 You might need to grant permissions to run the file depending on your operating system and settings.
 
+[NOTE]
+====
+If you are using macOS, you might encounter the error `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overzealous approach towards security by macOS and could be fixed by moving the "KIE Sandbox Extended Services" application out of quarantine. An example command to do that is `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app`. Please refer this https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/[article] for more details around this. 
```

Comment:
```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. For more information, see https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/[article].
```

Reply:
I personally would vote to don't link solutions from articles that are not official from apple or other trusted sources on our official docs. Maybe we can write similar content from the website in our docs.

The real solution for this would be to sign our binaries.